### PR TITLE
UIF-90 Update Fiscal year record view

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
Update Fiscal year record view
https://issues.folio.org/browse/UIF-90

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/FiscalYear/FiscalYearsList/FiscalYearsList.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        initialResultCount={INITIAL_RESULT_COUNT}
+        resultCountIncrement={RESULT_COUNT_INCREMENT}
+        viewRecordComponent={FiscalYearDetails}
+        visibleColumns={visibleColumns}
+        columnMapping={columnMapping}
+        viewRecordPerms="finance-storage.fiscal-years.item.get"
+        newRecordPerms="finance-storage.fiscal-years.item.post"
+        parentResources={resources}
+        parentMutator={mutator}
+        stripes={stripes}
```
Comment (aliaksei-chumakou) [inline src/components/FiscalYear/FiscalYearsList/FiscalYearsList.js]: it's redundant

Comment (AndreiShumski2) [reply src/components/FiscalYear/FiscalYearsList/FiscalYearsList.js]: yep. fixed

File: src/components/FiscalYear/FiscalYearDetails/FiscalYearDetails.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    ({ onToggle }) => (
+      <MenuSection id="fiscal-year-details-actions">
+        <DetailsEditAction
+          perm="finance-storage.fiscal-years.item.put"
+          onEdit={onEdit}
+          toggleActionMenu={onToggle}
+        />
+      </MenuSection>
+    ),
+    [],
```
Comment (NikitaSedyx) [inline src/components/FiscalYear/FiscalYearDetails/FiscalYearDetails.js]: `onEdit` ?

Comment (AndreiShumski2) [reply src/components/FiscalYear/FiscalYearDetails/FiscalYearDetails.js]: `FiscalYearForm` is disconnected due to BE schema changes

Comment (NikitaSedyx) [reply src/components/FiscalYear/FiscalYearDetails/FiscalYearDetails.js]: I still don't think this array should be mepty

File: src/components/FiscalYear/FiscalYearDetails/FiscalYearDetails.js [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+      <MenuSection id="fiscal-year-details-actions">
+        <DetailsEditAction
+          perm="finance-storage.fiscal-years.item.put"
+          onEdit={onEdit}
+          toggleActionMenu={onToggle}
+        />
+        <DetailsRemoveAction
+          perm="finance-storage.fiscal-years.item.delete"
+          onRemove={onRemove}
+          toggleActionMenu={toggleRemoveConfirmation}
```
Comment (NikitaSedyx) [inline src/components/FiscalYear/FiscalYearDetails/FiscalYearDetails.js]: does it work?

Comment (AndreiShumski2) [reply src/components/FiscalYear/FiscalYearDetails/FiscalYearDetails.js]: fixed)

**Pull request merged**